go/vt/automation/testing_emit_echo_task.go

```diff
@@ -0,0 +1,19 @@
+// Copyright 2015, Google Inc. All rights reserved.
```

Comment:
DONE

Shengzhe, thanks for the example! The ability to set functions so easily was the missing piece :-)

I've moved all code (including the two test tasks) into a new file "testutils_test.go".